gut_to_soil_manuscript_figures/_methods.py

```diff
@@ -80,6 +79,13 @@ def pcoa_2d(output_dir: str, metadata: qiime2.Metadata,
         <body>
             <h1>2D PCoA Plot</h1>
             <img src="pcoa_plot.png" alt="PCoA Plot">
+        ''')
+        if export_legend:
```

Comment:
```suggestion
        if export_legend:
            legend_fp = os.path.join(output_dir, 'legend.png')
```

Reply:
Now this functions essentially the same (after it's cast to a string) - this also matches the other checks I have in place in the actual script itself.